src/games/gameitem.py

```diff
@@ -294,70 +292,66 @@ def update(self, message, client):
-
-                    if i == 1 :
-                        displayPlayer = ("<td align = 'left' valign='center' width = '150'>%s</td>" % playerStr)
-                    elif i == self.nTeams :
-                        displayPlayer = ("<td align = 'right' valign='center' width = '150'>%s</td>" % playerStr)
-                    else :
-                        displayPlayer = ("<td align = 'center' valign='center' width = '150'>%s</td>" % playerStr)
+                    if player.rating_deviation < 200:
+                        playerStr += " (%s)" % str(player.rating_estimate())
 
                     country = os.path.join(util.COMMON_DIR, "chat/countries/%s.png" % (player.country or '').lower())
 
-                    if i == self.nTeams :
-                        displayPlayer += '<td width="16"><img src = "'+country+'" width="16" height="16"></td>'
-                    else :
-                        displayPlayer = '<td width="16"><img src = "'+country+'" width="16" height="16"></td>' + displayPlayer
+                    if i == 1:
+                        player_tr = "<tr><td><img src='%s'></td>" \
+                                            "<td align='left' valign='middle' width='135'>%s</td></tr>" % (country, playerStr)
+                    elif i == self.nTeams:
+                        player_tr = "<tr><td align='right' valign='middle' width='135'>%s</td>" \
+                                            "<td><img src='%s'></td></tr>" % (playerStr, country)
+                    else:
+                        player_tr = "<tr><td><img src='%s'></td>" \
+                                            "<td align='center' valign='middle' width='135'>%s</td></tr>" % (country, playerStr)
 
-                    display = ("<tr>%s</tr>" % displayPlayer)
-                    teamDisplay.append(display)
-                        
-                members = "".join(teamDisplay)
-                
-                teamlist.append("<td>" +teamtxt + members + "</table></td>")
-            else :
+                    teamplayer.append(player_tr)
+
+                teamplayer.append("</table></td>")
+                members = "".join(teamplayer)
+
+                teamlist.append(members)
+            else:
                 observerlist.append(",".join(self.teams[team]))
 
-        teams_string += "<td valign='center' height='100%'><font valign='center' color='black' size='+5'>VS</font></td>".join(teamlist)
+        teams_string += "<td valign='middle' height='100%'><font color='black' size='+5'>VS</font></td>".join(teamlist)
 
         observers = ""
-        if len(observerlist) != 0 :
+        if len(observerlist) != 0:
             observers = "Observers : "
             observers += ",".join(observerlist)        
 
-        mods = ""
-
         if self.mods:
-            mods += "<br/>With " + "<br/>".join(self.mods.values())
+            mods = "<br />With: " + "<br />".join(self.mods.values())
+        else:
```

Comment:
Slower in what sense?

It's python's version of the ternary conditional operator, in e.g. C-like languages:
```javascript
var mods = mods ? mods.values() : "";
```